src/network/netdev/tuntap.c

```diff
@@ -161,4 +162,5 @@ const NetDevVTable tap_vtable = {
         .done = tuntap_done,
         .create = netdev_create_tuntap,
         .create_type = NETDEV_CREATE_INDEPENDENT,
+        .do_not_generate_mac = true,
```

Comment:
Please drop this. we can set mac for tap device.

Reply:
MACAddress= for tun or tap devices are set by .network file, not .netdev file. So, I think the generated mac address is simply ignored. No? Am I missing something?

Reply:
Yes then it's not required at all. By default it's false no ?

Reply:
Ah, I see. Will update.